tests/restore/table/test_table_restore_service.py

```diff
@@ -0,0 +1,51 @@
+import unittest
+import uuid
+
+from mock import Mock, MagicMock, patch
+
+from src.restore.list.backup_list_restore_service import \
+    BackupListRestoreService, BackupListRestoreRequest, BackupItem
+from src.restore.status.restoration_job_status_service import \
+    RestorationJobStatusService
+from src.restore.table.backup_finder import BackupFinder
+from src.restore.table.table_restore_service import TableRestoreService
+
+
+class TestTableRestoreService(unittest.TestCase):
+
+    @patch.object(BackupListRestoreService, 'restore')
+    @patch.object(BackupFinder, 'for_table')
+    @patch.object(uuid, 'uuid4', return_value=123)
+    def test_happy_path(self, _, for_table, restore):
```

Comment:
I don;t like "test_happy_path" name, I want sth which describe what should happen

Reply:
Okay, I did chang the name to 'test_service_call_backup_list_restore_with_one_item_only'.
And removed redundant checks, because it should test only that redirects to backup list restore service.